### ISSUE TITLE
usbip_vhci not loading on Windows 7

### ISSUE THREAD

**Issue by tatokis** [cezuni/usbip-win#12]
I installed the `usbip-win 0.0.4` pre-release on Windows 7 (7601), and even after manually generating a signed cat file, windows refuses to load the driver with `Code 39`.

Event viewer says 
```
Application popup:  : \SystemRoot\system32\DRIVERS\usbip_vhci.sys cannot find ntoskrnl.exe KeInitializeSpinLock
```
which looks like Win7 compatibility was broken.

I tried compiling the driver with the old build system and the Win7 DDK but there were quite a few issues and I could not get it to compile successfully.

I do not have access to a proper Windows 10 dev installation at the moment to have a look, so it would be appreciated if this could be fixed.

**Comment by cezuni** [comment]
This issue seems to be duplicate of #8. Anyway, it is almost obvious that usbip-win is not working on win7. But in my opinion, win7 support cannot be a high priority. Win10 is already a major version and there are so many malfunctioning and unstable features even on win10. 😢

**Comment by tatokis** [comment]
Thank you for the response.
I took the time to install VS2017 and the Win 10 WDK.

I changed the project's Target OS Version in VS to Windows 7, and then built the driver.
![79976dc9aab7](https://user-images.githubusercontent.com/6003656/52771977-07e50300-3040-11e9-8589-134968b25fc6.jpg)


Windows 7 was then able to successfully load my newly built driver.
![usbipwin_](https://user-images.githubusercontent.com/6003656/52771812-94db8c80-303f-11e9-8c71-0f8b93e4785a.png)

I have attached x64 debug binaries built from https://github.com/cezuni/usbip-win/commit/94b35a0916962874f062076176ec597a612aaa77 for anyone wanting to try to run this.
[usbipwinx64.zip](https://github.com/cezuni/usbip-win/files/2863924/usbipwinx64.zip)
